bin/rdme

```diff
@@ -1,7 +1,7 @@
 #! /usr/bin/env node
```

Comment:
I'm assuming it's fine that this file doesn't have an extension since we have this line?

Reply:
yep